systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -73,31 +73,31 @@ public static String waitTopicHasRolled(final String namespaceName, String topic
     }
 
     public static void waitForKafkaTopicCreation(String namespaceName, String topicName) {
-        LOGGER.info("Waiting for KafkaTopic {} creation ", topicName);
-        TestUtils.waitFor("KafkaTopic creation " + topicName, Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, READINESS_TIMEOUT,
+        LOGGER.info("Waiting for KafkaTopic: {} creation ", topicName);
+        TestUtils.waitFor("Kafka Topic creation " + topicName, Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS, READINESS_TIMEOUT,
```

Comment:
One line above you have `KafkaTopic` - it's still the CR that we are waiting for, so I would keep it together